src/supermarket/app/views/cookbooks/_main.html.erb

```diff
@@ -112,14 +112,20 @@
     <% end %>
     <% if ROLLOUT.active?(:fieri) %>
       <div class="content" id="quality">
-        <% if version.foodcritic_failure.nil? %>
-          <h2>No quality metric results found.</h2>
-        <% else %>
+        <% if version.foodcritic_failure.present? || version.collaborator_failure.present? %>
```

Comment:
Make sure that this logic is consistent with the working logic in master re: double negative with the foodcritic failure and three states of a foodcritic response: `nil`, `false`, and `not false`.